app/helpers/home_helper.rb

```diff
@@ -0,0 +1,2 @@
+module HomeHelper
```

Comment:
This is unused so we could get rid of it. Guessing Rails generated it.

Reply:
Yeah, not sure what those do. There are three of them, all doing nothing.  I'd prefer to just leave them for now and then look again when we do some tidying up.